README.md

```diff
@@ -1,47 +1,185 @@
 # react-e [![CircleCI](https://circleci.com/gh/Originate/react-e.svg?style=svg)](https://circleci.com/gh/Originate/react-e)
 
-React helper to generate elements
+JSX replacement that doesn't need to be compiled.
 
-## Installing
-`npm install --save react-e`
+## Basic Usage
 
-## Example Usage
+```javascript
+e = require('react-e')
 
-```coffeescript
-e = require 'react-e'
-
-e '.container',
-  e 'button.primary', 'Click me'
-  e 'span.text', className: active: true, 'Some text'
+e(Modal,
+  e('.container',
```

Comment:
In your previous example it would have to be:
```
{$, span} = require('react-e').domElements
// or $ = require('react-e') for the old way

$(Modal,
  $('.container',
    span('.text', {className: {active: this.state.active}}, 'Some text'),
    span({className: ['text', {active: this.state.active}]}, 'Some text'),  // alternative syntax
    $(Button, {className: 'primary'}, 'Close')
  )
)
```
and since probably 90% of the elements used are going to be `div`s anyway, there's no real advantage to exposing the alternate syntax for the 10% of elements that aren't going to be divs